refactor(ui): migrate table to div

wa0x6e opened the pull request:
### Summary

<!-- Related issues, a description or list of the changes and the motivation behind them -->

Closes: #339 #330 #145

Migrate tables to div for pages:

- votes 
- delegates
- leaderboard

Migrating to div will fix a few issues with table

### How to test

1. Go to a proposal votes page
2. It should display data as before
3. The voting power background should display correctly on webkit engine (https://github.com/snapshot-labs/sx-monorepo/issues/145)
4. Go to the delegates and leaderboard page
5. It should display data as before
6. The 1px gap described in https://github.com/snapshot-labs/sx-monorepo/issues/330 should be fixed

Sekhmet commented:
<img width="242" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/67c1d450-c7fd-4e0b-93ac-8e86bea64f3b">

Can we align headers to rows? Right now headers are a bit more to the right than the values.

wa0x6e commented:
> <img alt="image" width="242" src="https://private-user-images.githubusercontent.com/1968722/330864322-67c1d450-c7fd-4e0b-93ac-8e86bea64f3b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU3OTA1MjYsIm5iZiI6MTcxNTc5MDIyNiwicGF0aCI6Ii8xOTY4NzIyLzMzMDg2NDMyMi02N2MxZDQ1MC1jN2ZkLTRlMGItOTNhYy04ZTg2YmVhNjRmM2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTVUMTYyMzQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjM5NjI0Y2NjYTg4NmNhYmZiOTg4YTIxNjk2ODcwOGQ0OTFiMTQyNzM2NDA2OTNhZTdmMTUxMmUzNzA1N2FmYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.bRED4EEpc8Is1aCoTtFNzeNBN16yKaBnUrH0l7gcW4Y">
> Can we align headers to rows? Right now headers are a bit more to the right than the values.
> 
> Looks like it happens not on all spaces so there it is: http://localhost:8080/#/s:cakevote.eth/proposal/0x41feef37054b6c090a2629d6eb2ed39f426f169b725cb565903e6b3ba1771bcc/votes

Fixed. Also added prevented sort arrow icons from shrinking

Sekhmet commented:
@wa0x6e looks like you forgot to push changes?

Sekhmet commented:
Also, do you know if we can do something about uneven columns when going smaller?

<img width="435" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/f62e18c1-cb26-4a8b-bad4-41f999e6311e">

Choice column seems to be moving around from row to row.

wa0x6e commented:
Yes, it's fixed in the commit, but I can not find it anymore ><"

wa0x6e commented:
Should be good now

Sekhmet reviewed (approved):
tACK

Looks good to me.
